src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -12,3 +12,7 @@ com.sonatype.nexus.api.iq.PolicyFact
 com.sonatype.nexus.api.iq.ComponentFact
 com.sonatype.nexus.api.iq.ConstraintFact
 com.sonatype.nexus.api.iq.ConditionFact
+com.sonatype.nexus.api.iq.ComponentIdentifier
+com.sonatype.nexus.api.iq.ComponentDisplayName
+com.sonatype.nexus.api.iq.ComponentDisplayNamePart
+com.sonatype.nexus.api.iq.Action
```

Comment:
They found more instances of problems that didn't surface in all the initial testing for this :cry: 
See comment below where I was able to replicate.

Reply:
shouldn't these be coming in from an upstream merge ?

Reply:
Went with this approach for this particular PR because of the mess that is JEP-200 - https://docs.sonatype.com/display/INT/Jenkins+Plugin+Development+Process#JenkinsPluginDevelopmentProcess-Reviewagainstthedownstreamrepositoryfirst,thentheupstream

Will re-create PR upstream once we're good here